eng/scripts/Generate-APIView-CodeFile.ps1

```diff
@@ -11,12 +11,13 @@ if (!(Test-Path -Path $ArtifactPath))
   exit 1
 }
 
-$apiviewParser = "@azure-tools/ts-genapi@1.0.3"
+$apiviewParser = "@azure-tools/ts-genapi@latest"
```

Comment:
In general floating to the latest like this tends to break things. We may need to consider an update strategy for this at some point.